test/unit/mdc-textfield/foundation.test.js

```diff
@@ -87,10 +87,10 @@ test('#setDisabled adds mdc-textfield--disabled when set to true', () => {
   td.verify(mockAdapter.addClass(cssClasses.DISABLED));
 });
 
-test('#setDisabled removes mdc-textfield--disabled when set to false', () => {
+test('#setDisabled sets icon tabindex to -1 when set to true', () => {
```

Comment:
Add a test for the opposite (to match the fix to the issue I noted above)

Reply:
Also, I'm confused why it looks like an existing test was changed/repurposed for this rather than adding a new one?

Reply:
That might just be the way the diff shook out. The test for `#setDisabled removes mdc-textfield--disabled when set to false` is still in there